app/services/OphanService.scala

```diff
@@ -0,0 +1,13 @@
+package services
+
+import com.gu.okhttp.RequestRunners
+
+object OphanService {
+
+  def apply(isTestService: Boolean): com.gu.acquisition.services.OphanService =
+    if (isTestService) {
+      com.gu.acquisition.services.MockOphanService
+    } else {
+      com.gu.acquisition.services.OphanService.prod(RequestRunners.client)
```

Comment:
I don't think we want to create a new Ophan service per non-test service request.

Reply:
agreed, we did have some issues with leaking thread pools when creating services more than once, so be very careful with that.
I think we have a cloudwatch metric for thread-count, so please check that for a few days after deploying this